workspace/compression_sandbox/cedrlang/cedrlang.py

```
@@ -132,21 +167,78 @@
+        children = []
+        for char in chars:
+            sub = _trie_to_regex(node[char])
+            if sub:
+                children.append(re.escape(char) + sub)
+            else:
+                children.append(re.escape(char))
+
+        if len(children) == 1:
+            res = children[0]
+        else:
+            res = "(?:" + "|".join(children) + ")"
```

Ruff (0.16.2)

[warning] Use ternary operator `res = children[0] if len(children) == 1 else "(?:" + "|".join(children) + ")"` instead of `if`-`else`-block

Replace `if`-`else`-block with `res = children[0] if len(children) == 1 else "(?:" + "|".join(children) + ")"`

(SIM108)